hoomd/mpcd/integrate.py

```diff
@@ -129,6 +149,21 @@ def cell_list(self):
         """
         return self._cell_list
 
+    @property
+    def virtual_particle_fillers(self):
+        return self._virtual_particle_fillers
+
+    @virtual_particle_fillers.setter
+    def virtual_particle_fillers(self, value):
+        _set_synced_list(self._virtual_particle_fillers, value)
+
+    @property
+    def _children(self):
```

Comment:
I couldn't really figure out from the code / docs: what is this property, and what I supposed to go in it? Did I override this right?

Reply:
`Operation._children` appears to be unused. It may be some previous attempt at child attach/detach management that we did not fully remove.

Reply:
Gotcha! So, should I remove this override here, and then someone can remove that code path in a separate PR?

Reply:
Yes.